hasher-matcher-actioner/hmalib/common/classification_models.py

```diff
@@ -16,10 +16,21 @@ class Label:
     key: str
     value: str
 
+    def __eq__(self, other) -> bool:
+        # Concrete type doesnt matter for equality, only key, value
+        # eg the following are equal:
+        #   ClassificationLabel("Classification", "true_positive")
+        #   Label("Classification", "true_positive")
+        return (
+            isinstance(other, Label)
+            and self.key == other.key
+            and self.value == other.value
+        )
+
 
 @dataclass(unsafe_hash=True)
 class ClassificationLabel(Label):
-    key: str = field(default="Classification", init=False)
```

Comment:
Need this because it was overriding child classes when deserialized 

eg.
ClassificationLabel("BankIDClassification", "bank 4") -> Label("Classification", "bank 4")